draft-ietf-dnsop-domain-verification-techniques.md

```diff
@@ -200,6 +200,30 @@ Other techniques such as email or HTTP(S) based validation are out-of-scope.
 
 * `Random Token`: a random value that uniquely identifies the DNS domain control validation challenge, defined in {{random-token}}.
 
+# Purpose of Domain Control Validation {#purpose}
+
+Domain Control Validation (DCV) is a challenge-response protocol that allows one party (the User) to prove operational control of the contents of a domain to an Application Service Provider (ASP).  This proof applies at a single point in time; it does not provide evidence of continuing control prior to the challenge or after the record is published.
+
+DCV is appropriate when the ASP is about to take a brief, time-bounded action that is potentially dangerous but not more harmful than the damage that could be done by malicious modification to the DNS zone.
+
+Appropriate uses of DCV:
+
+* Issuing a time-bounded TLS certificate via an ACME "dns-01" challenge.
+  - Control of the DNS zone would be sufficient to acquire a certificate anyway via the ACME "http-01" challenge.
+* Allowing a user to claim ownership of a domain name in a third-party system for a limited period of time.
+  - This creates a risk of impersonation, but the User has already demonstrated their ability to impersonate this name.
+* Sending a large batch of TLS authenticated requests to a server using this name.
+  - This creates a risk of outage due to overload, but the User could create an outage anyway by deleting important records.
+
+Inappropriate uses of DCV:
+
+* Allowing the User to claim ownership of the domain name indefinitely in a third-party system.
+  - The User could be an attacker who has only transient control of the domain name, or the domain name could be transferred to a new owner.
```

Comment:
What is the practical risk of this when combined with another delegation such as a CNAME to the ASP?  The CNAME is what delegates to the ASP and the DCV entry is what demonstrates which of the ASP's customers/accounts has control.  How practical is this "stale control" risk?  Would leaving the DCV entry in-place while still using the service suffice (which means we'd want to update some of the "removal" guidance?  The assumption that a new future owner of a zone would remove or not create the DCV entry and if they wanted to use the ASP they'd revalidate there (so perhaps some guidance to the effect of this use-case). At some point the operational burden of redoing DCV at some frequency will overweigh its value and it won't get implemented at all.